### ISSUE TITLE
Parsing failed: mapping values are not allowed here

### ISSUE THREAD

**Issue by herrriehm** [thp/urlwatch#660]
So I want to urlwatch some company's career site. In a first curl request I get a token, and in a second curl request I use this token to pull the job offerings. The following chain of commands works perfectly fine on my command line:

`token=$(curl --silent 'https://mycompany.csod.com/...' 2>&1 | grep -Po '[a-zA-Z0-9\._-]{300}') && curl 'https://api.csod.com/...' -H 'Authorization: Bearer '$token -H 'Content-Type: application/json' -H 'TE: trailers' --data-raw '{...}'`

Then I put this command into urlwatch as following:

```
kind: shell
command: token=$(curl --silent 'https://mycompany.csod.com/...' 2>&1 | grep -Po '[a-zA-Z0-9\._-]{300}') && curl 'https://api.csod.com/...' -H 'Authorization: Bearer '$token -H 'Content-Type: application/json' -H 'TE: trailers' --data-raw '{...}'
```

However, upon saving, urlwatch complains with:

```
Parsing failed:
======
mapping values are not allowed here
  in "/home/user/.urlwatch/urls.edit.yaml", line 1, column 148
======
```

On position 1/148 there is the colon behind `-H 'Authorization`. If I delete this header, the error goes on to the next colon behind `Content-Type`. 

Is urlwatch mis-parsing the colons as yaml commands? Or am I missing something? Thank you very much for your help.

**Comment by kongomongo** [comment]
looks like a quoting issue.

yaml expects 

command: "stuff"

either put all your stuff in double quotes or swap single with double quotes and put single quotes around all

**Comment by herrriehm** [comment]
Thank you @kongomongo, you nudged me in the right direction. I got it working.

I was wondering why urlwatch did not complain about the colon in `https://mycompany`, but the one in `Authorization: Bearer`. So I just deleted the space after the colon – and it worked.

Maybe urlwatch parses colon-space as a yaml key-value pair, no matter its position in a string?

Closing for now, but maybe @thp could take a look.
